### DISCUSSION TITLE
Default webpage Fonts colors

### DISCUSSION THREAD

**Discussion by patata3000** [qutebrowser/qutebrowser#8107] [Q&A] [answered]
I cannot find any default way to set a default font color for webpage. We have a `colors.webpage.bg` but nothing for foreground. So for some website, where I use a dark background, I cannot set a light foreground. Is there a workaround or am I doomed?

**Comment by toofar** [comment, marked as answer]
You can point the `content.user_stylesheets` setting at one or more CSS files that can be used to style websites.

**Reply by patata3000** [reply]
More work haha. I guess this won't be for now. Thanks
